lib/Errors/ReloadedError.js

```diff
@@ -0,0 +1,18 @@
+const Messages = require("./Messages");
+
+/**
+ * A generic error from the library.
+ * @extends Error
+ * @private
+ */
+class ReloadedError extends Error {
+    constructor(code, type = "LIBRARY") {
```

Comment:
```suggestion
    constructor(code, type = "RELOADED") {
```
`RELOADED` will sound better.

Reply:
I don't see any difference but ok